### ISSUE TITLE
Do not allow roles below editor

### ISSUE THREAD

**Issue by sinabahram** [coyote-team/coyote-wp#91]
The plugin should not proceed if it does not receive at least an editor role for the token,. If it does, then someone fixes it by upgrading the author account to an editor, it will then appear to work, but the plugin will not go create the wordpress resource group with corresponding callback URL, leading to a bunch of orphaned resources.

**Comment by jkva** [comment]
The available roles available within Coyote:

* guest
* viewer
* author
* editor
* admin
* owner

editor, admin and owner are valid roles in this context.
